tests/program_analysis/test_py_ast_to_cast.py

```diff
@@ -0,0 +1,846 @@
 "Number", "number": 2, "source_refs": [{"col_end": 18, "col_start": 17, "node_type": "SourceRef", "row_end": 4, "row_start": 4, "source_file_name": "test_class_1.py"}]}, "source_refs": [{"col_end": 18, "col_start": 8, "node_type": "SourceRef", "row_end": 4, "row_start": 4, "source_file_name": "test_class_1.py"}]}], "func_args": [{"node_type": "Var", "source_refs": [{"col_end": 21, "col_start": 17, "node_type": "SourceRef", "row_end": 2, "row_start": 2, "source_file_name": "test_class_1.py"}], "type": "integer", "val": {"name": "self", "node_type": "Name", "source_refs": [{"col_end": 21, "col_start": 17, "node_type": "SourceRef", "row_end": 2, "row_start": 2, "source_file_name": "test_class_1.py"}]}}], "name": "__init__", "node_type": "FunctionDef", "source_refs": [{"col_end": 18, "col_start": 4, "node_type": "SourceRef", "row_end": 4, "row_start": 2, "source_file_name": "test_class_1.py"}]}], "name": "Foo", "node_type": "ClassDef", "source_refs": [{"col_end": 18, "col_start": 0, "node_type": "SourceRef", "row_end": 4, "row_start": 1, "source_file_name": "test_class_1.py"}]}, {"body": [{"left": {"node_type": "Var", "source_refs": [{"col_end": 5, "col_start": 4, "node_type": "SourceRef", "row_end": 7, "row_start": 7, "source_file_name": "test_class_1.py"}], "type": "integer", "val": {"name": "f", "node_type": "Name", "source_refs": [{"col_end": 5, "col_start": 4, "node_type": "SourceRef", "row_end": 7, "row_start": 7, "source_file_name": "test_class_1.py"}]}}, "node_type": "Assignment", "right": {"arguments": [], "func": {"name": "Foo", "node_type": "Name", "source_refs": [{"col_end": 13, "col_start": 8, "node_type": "SourceRef", "row_end": 7, "row_start": 7, "source_file_name": "test_class_1.py"}]}, "node_type": "Call", "source_refs": [{"col_end": 13, "col_start": 8, "node_type": "SourceRef", "row_end": 7, "row_start": 7, "source_file_name": "test_class_1.py"}]}, "source_refs": [{"col_end": 13, "col_start": 4, "node_type": "SourceRef", "row_end": 7, "row_start": 7, "source_file_name": "test_class_1.py"}]}], "func_args": [], "name": "main", "node_type": "FunctionDef", "source_refs": [{"col_end": 13, "col_start": 0, "node_type": "SourceRef", "row_end": 7, "row_start": 6, "source_file_name": "test_class_1.py"}]}, {"expr": {"arguments": [], "func": {"name": "main", "node_type": "Name", "source_refs": [{"col_end": 6, "col_start": 0, "node_type": "SourceRef", "row_end": 9, "row_start": 9, "source_file_name": "test_class_1.py"}]}, "node_type": "Call", "source_refs": [{"col_end": 6, "col_start": 0, "node_type": "SourceRef", "row_end": 9, "row_start": 9, "source_file_name": "test_class_1.py"}]}, "node_type": "Expr", "source_refs": [{"col_end": 6, "col_start": 0, "node_type": "SourceRef", "row_end": 9, "row_start": 9, "source_file_name": "test_class_1.py"}]}], "name": "test_class_1", "node_type": "Module", "source_refs": [{"col_end": null, "col_start": null, "node_type": "SourceRef", "row_end": 9, "row_start": 1, "source_file_name": "test_class_1.py"}]}]}'
```

Comment:
Yes! I can do that! I was trying to think of a better way to hold the test data that wasn't just hardcoding all the data in each of the files. This will work! Thank you! I'll work on this and put an acknowledgment comment on the other conversations that are about this topic as well.